tests/integration/__init__.py

```diff
@@ -405,7 +405,7 @@ def _id_and_mark(f):
 notdse = unittest.skipIf(DSE_VERSION, "DSE not supported")
 requiredse = unittest.skipUnless(DSE_VERSION, "DSE required")
 requirescloudproxy = unittest.skipIf(CLOUD_PROXY_PATH is None, "Cloud Proxy path hasn't been specified")
-
+notscylla = unittest.skipIf(SCYLLA_VERSION, "Does not support scylla")
```

Comment:
You added this attribute, and imported it in test file, but did not use it. If its not necessary, can we remove it?

In general, if we don't want to run something on Scylla (because Scylla doesn't have some functionality) then I'd prefer to use `xfail_scylla`. This is because we'll get notified (by failing CI) that Scylla got some functionality and we should change the annotation. Otherwise noone will ever do it.

Reply:
ok, removed.